tests/system_health/test_watchdog.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import pytest
+import time
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('any')
+]
+
+logger = logging.getLogger(__name__)
+
+SLEEP_TIME = 10
+
+
+@pytest.fixture
+def pause_orchagent(duthost):
+    # find orchagent pid
+    pid = duthost.shell(
+                    r"ps -ef | grep orchagent | grep -v grep | awk '{print $2}'",
```

Comment:
>ps -ef | grep orchagent | grep [](http://example.com/codeflow?start=22&length=30)

Not a blocking issue, you may consider pgrep.

Reply:
Fixed.